dev/ci-velox-buildstatic-centos-7.sh

```diff
@@ -2,12 +2,9 @@
 
 set -e
 
-yum install sudo patch java-1.8.0-openjdk-devel -y
```

Comment:
Initially this is kept in case someone is using this script from a clean centos 7 image

Reply:
@zhouyuan, maybe we can just tell user to refer weekly build job for building from clean image. centos-7/8/9, ubuntu-20.04/22.04 are covered in that job. I will document this guide. Thanks!